ci/analyze.sh

```diff
@@ -30,7 +30,7 @@ function follow_links() (
 SCRIPT_DIR=$(follow_links "$(dirname -- "${BASH_SOURCE[0]}")")
 SRC_DIR="$(cd "$SCRIPT_DIR/../.."; pwd -P)"
 FLUTTER_DIR="$SRC_DIR/flutter"
-DART_BIN="$SRC_DIR/third_party/dart/tools/sdks/dart-sdk/bin"
+DART_BIN="$SRC_DIR/out/host_debug_unopt/dart-sdk/bin"
```

Comment:
Check that this exists, and fail with an error message that gives some build instructions, like:
```
$ gn --runtime-mode debug --unopt --no-lto
$ ninja -C out/host_debug_unopt
```

Reply:
Gotcha. I have `'gn --unoptimized --runtime-mode=debug'` from the compiling-the-engine wiki, but am seeing:

```
ERROR No command specified.
Most commonly you want "gn gen <out_dir>" to make a build dir.
```

Is there a build target I should specify for the gn command?

Reply:
Oh, sorry. The engine tree has it's own script that wraps `gn`. It's under `flutter/tools/gn`.